PixivModelFanbox.py

```diff
@@ -434,11 +450,7 @@ def WriteHtml(self, html_template, useAbsolutePaths, filename):
 
         cover_image = ""
         if self.coverImageUrl:
-            # Issue #930
-            if self.fullCoverSizeUrl:
-                cover_image = f'<div class="cover"><a href="{self.fullCoverSizeUrl}" target="_blank"><img style="max-width: 1200px; height:auto;" src="{self.coverImageUrl}"/></a></div>'
-            else:
-                cover_image = f'<div class="cover"><img src="{self.coverImageUrl}"/></div>'
+            cover_image = f'<div class="cover"><img src="{self.coverImageUrl}"/></div>'
```

Comment:
need to adjust the cover size, else it will be too big in the html. That is why I add inline style here.

Reply:
i tested with default html template, and `%coverImage%` is placed under a `<div class="main">` which is under a `<div class="root">`, and in css there are these parts: `.root{max-width:1280px; margin:auto; background-color:#ffffff;}` and `*{margin:0px; padding:0px; max-width:100%; overflow:auto;}`, so the max-width is set to 1280px....
unless the user uses a custom template, in which case the user could modify the style by setting the styles of class "cover" for this `div` and tokens under it.
putting the max-width in the token made it impossible to change it without changing the codes, so i removed it.

also the `href` inside `a` token and the `src` inside `img` token are actually set the same, so i just removed the `a` token wrapping outside the `img` token.

that's my opinion though.

Reply:
gotcha

Reply:
glad we are on the same page
;P